server/app/models/blog.js

```diff
@@ -5,7 +5,7 @@ var sql = require('../../config/sequelize')();
 var BlogList = sql.define('blogs', {
   title: {type: Sequelize.TEXT, allowNull: false},
   content: {type: Sequelize.TEXT, allowNull: false},
-  userId: {type: Sequelize.INTEGER, allowNull: false},
+  userId: {type: Sequelize.TEXT, allowNull: false},
```

Comment:
Please, rename this to `owner`. It will help some of our functions that checks for ownership.